### ISSUE TITLE
Update the selected room menu

### ISSUE THREAD

**Issue by giomfo** [tchapgouv/tchap-android-v2#78]
Currently in the tchap v2, a long press on a room in the rooms list opens the following menu:
![image](https://user-images.githubusercontent.com/8969772/118551789-61c45e00-b75e-11eb-8bd2-a0b39ec0a108.png)

- the notifications mode will be handled by #77 
- we will remove the low priority option: 
![image](https://user-images.githubusercontent.com/8969772/118551921-8d474880-b75e-11eb-946b-af694ae47509.png)
- we will replace the favourite icon 
![image](https://user-images.githubusercontent.com/8969772/118552005-a4863600-b75e-11eb-9f45-b1ed74367509.png) with the pin icon 
![image](https://user-images.githubusercontent.com/8969772/118552039-b36ce880-b75e-11eb-82bf-e3117670ce93.png) the 2 states of the pin are available in design v1 ( empty pin by default, filled pin if the room is pinned
- a barred bell is displayed after the room name when the notifications are mute
![image](https://user-images.githubusercontent.com/8969772/118552772-ad2b3c00-b75f-11eb-88f7-8cc4c1814b82.png)

**Comment by giomfo** [comment]
blocked by #77

**Comment by giomfo** [comment]
the last point relative to the muted room will be handled later with #99
